app-modules/inventory-management/database/migrations/2023_12_27_175512_create_maintenance_activities_table.php

```diff
@@ -44,16 +44,15 @@ public function up(): void
     {
         Schema::create('maintenance_activities', function (Blueprint $table) {
             $table->uuid('id')->primary();
+
             $table->foreignUuid('asset_id')->constrained('assets');
-            // It's possible that the maintenance provider can be an already known entity within the system as well, like a user
-            // For now, we will assume that it is a separate and new entity specifically for this purpose.
             $table->foreignUuid('maintenance_provider_id')->nullable()->constrained('maintenance_providers');
-            // We might want to standardize the details of a maintenance activity, so it can be easily replicated with consistency
```

Comment:
Did you create tickets for some of the comments throughout migrations?

Reply:
I did, any of the comments I removed I went and created tickets for.